src/kubectlUtils.ts

```diff
@@ -101,6 +110,50 @@ export async function getNamespaces(kubectl: Kubectl): Promise<Namespace[]> {
     });
 }
 
+export async function getDeployments(kubectl: Kubectl): Promise<Deployment[]> {
+    const currentNS = await currentNamespace(kubectl);
+
+    const shellResult = await kubectl.invokeAsync("get deployments -o json --namespace=" + currentNS);
+    if (shellResult.code !== 0) {
+        vscode.window.showErrorMessage(shellResult.stderr);
+        return [];
+    }
+    const depList = JSON.parse(shellResult.stdout);
+    return depList.items.map((item) => {
+        return {
+            name: item.metadata.name,
+            selector: item.spec.selector
+        };
+    });
+}
+
+export async function getPods(kubectl: Kubectl, selector: any): Promise<Pod[]> {
+    const currentNS = await currentNamespace(kubectl);
+
+    var labels = [];

```

Comment:
using `var` here (and once more below) is causing tslint to complain, and the travis build to fail. you can change the `var` declaration here to `const`, and the one below to a `let`.

Reply:
Done.